src/cantaloupe.js

```diff
@@ -0,0 +1,22 @@
+import qs from "query-string";
+import testdata from "./couch/testManifest.json";
+const cantaloupeUrl = process.env.CANTALOUPE;
+function testManifestData(
+  token,
+  itentifier,
+  region,
+  size,
+  rotation,
+  quality,
+  format
+) {
+  var Jdata = testdata;
+
+  var Obj = JSON.parse(JSON.stringify(Jdata));
+  let url = encodeURIComponent(
+    [cantaloupeUrl].join("/") + Obj.m0bc1df2gh3jk.slug + "/data/sip/data/files/"
+  );
+  console.log("URL : ", url);
+}
+
+export { testManifestData };
```

Comment:
So, for this you're going to need two methods: one here, and one in couch.js.

The one in couch.js imports the test data from the json file, and does the work to bring in the canvas paths from the canvas objects in the test data. I'm not sure what that will end up looking like, but it's a similar process to what will happen when you're pulling actual data from couch. This method will eventually replace `testManifest` when everything is hooked up.

The one here takes a Swift path, removes the `https://swift.canadiana.ca/v1/AUTH_crkn/repository/` from it, URL-escapes the rest, and then creates a Cantaloupe URL with that as the identifier. The only other IIIF parameter you'll need to pass is one relating to the size you're requesting, but for that you can probably just use a boolean called `isThumbnail`. Region, rotation, quality, and format will always be `full`, `0`, `default`, and `jpg`, respectively.

You're also doing a `JSON.parse(JSON.stringify(testdata))` call here, which in the end doesn't actually do anything.